proto/migration/v1alpha1_to_v1.go

```diff
@@ -380,6 +380,20 @@ func V1Alpha1BeaconBlockAltairToV2(v1alpha1Block *ethpbalpha.BeaconBlockAltair)
 	return v2Block, nil
 }
 
+// V1Alpha1BeaconBlockMergeToV2 converts a v1alpha1 Merge beacon block to a v2
```

Comment:
Can we either change the file name to something like `convert_block` or move this to a new file `v1alpha1_to_v2`?